websites/G/GetPocket/presence.ts

```diff
@@ -0,0 +1,83 @@
+const presence = new Presence({
+  clientId: "775661829887754260"
+});
+const browsingStamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  };
+
+  
+  const exploreTitles = document.querySelector("#__next > div > header") as HTMLHeadingElement;
+  const articleTitles = document.querySelector("#__next > div.p16bck5y > header > div > header > h1");
+  const newsPublisher = document.querySelector("#__next > div.p16bck5y > header > div > div.b12pz0kr > a");
+  const authorName = document.querySelector("#__next > div.p16bck5y > header > div > div.b12pz0kr > ul > li");
```

Comment:
merge consts

Reply:
I cannot. Everyone of those controls a different thing.

Reply:
Eeee, you can, merge consts.
for example:
```const var1 = var(),
var2 = var3();```

Reply:
Hey! I tried that and it messed everything up!

Reply:
So you do that wrong probably

Reply:
We got it fixed dw @ririxichan (cutie)